tests/src/test/scala/whisk/core/controller/test/WebActionsApiTests.scala

```diff
@@ -1711,6 +1711,19 @@ trait WebActionsApiBaseTests extends ControllerTestCommon with BeforeAndAfterEac
           }
         }
     }
+
+    it should s"allowed string based status code (auth? ${creds.isDefined})" in {
+      implicit val tid = transid()
+
+      Seq(s"$systemId/proxy/export_c.http").foreach { path =>
+        invocationsAllowed += 1
+        actionResult = Some(JsObject(webApiDirectives.statusCode -> JsString("200")))
+
+        Head(s"$testRoutePath/$path") ~> Route.seal(routes(creds)) ~> check {
```

Comment:
can you add a negative test also, where s/200/xyz/

Reply:
Sure, thanks!